issue_1/contracts/WordRegistry.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+import "hardhat/console.sol";
+
+import "./yield-utils-v2/contracts/access/Ownable.sol";
+
+/**
+@title Dictatorship-friendly word registry
+@author Owleksiy
+@notice This contract allows users to register/unregister (claim/unclaim) words
+Once a word is regiistered by a user, other users can't register it, unless the
+original user unregisters it.
+
+Additionally, the owner of the contract can expropriate any registered word
+ */
+contract WordRegistry is Ownable {
+    event OwnershipChange(string word, address previousOwner, address newOwner);
+    // word -> owner mapping
+    mapping(string => address) private word_to_owner;
+
+    constructor() {
+        console.log("Creating a registry, owner:", msg.sender);
+    }
+
+    function _is_owned_by_somebody(string memory word) private view returns (bool) {
+        return word_to_owner[word] != address(0);
+    }
+
+    function _transfer(
```

Comment:
There is a very well known `_transfer` function in ERC20.sol. I wouldn't reuse the name since it can lead to confusion.

Reply:
ack